src/applications/vaos/services/healthcare-service/transformers.js

```diff
@@ -159,7 +159,36 @@ export function findCharacteristic(clinic, characteristic) {
     return element.text === characteristic;
   });
 
-  return result.coding.code && result.coding.code !== undefined
-    ? result.coding.code
-    : result.coding.display;
+  if (result) {
```

Comment:
I think this block could be `return result?.coding?.code || result?.coding?.display;`

Reply:
Done